readium/shared/src/main/java/org/readium/r2/shared/util/Url.kt

```diff
@@ -359,7 +359,9 @@ private fun String.isValidUrl(): Boolean =
 @JvmInline
 public value class FileExtension(
     public val value: String
-)
+) {
+    override fun toString(): String = value
+}
```

Comment:
Isn't this implicit with value classes?

Reply:
Unfortunately no, it's similar to data classes: `FileExtension(value = "epub")`.

I used this to simplify stuff like: "filename.$fileExtension".